ember/app/components/maintainer-table/template.hbs

```diff
@@ -0,0 +1,28 @@
+<h2 class="table-header">Maintainers</h2>
+<hr class='seperator' />
+{{#if @maintainers}}
+  <table>
+    <thead>
+      <tr>
+        <th>Username </th>
+        <th>Email Address</th>
+      </tr>
+    </thead>
+    <tbody>
+      {{#each (sort-by "isPrimary" @maintainers) as |maintainer|}}
+      <tr>
+        <td>
+          {{maintainer.user.username}}
+        </td>
+        <td>
+          {{maintainer.user.email}}
+        </td>
+      </tr>
```

Comment:
Where can I see who the primary maintainer is?

Reply:
The first one in the table is the primary maintainer
https://github.com/adfinis/Outdated/blob/bc8148f87627ddec0afb74923f7b64f7567dc483/ember/app/components/maintainer-table/template.hbs#L11-L13

Reply:
Let's mark that one so it's more obvious. Maybe add like a star icon next to the name or something..

Reply:
The primary maintainer is now marked with a star icon
https://github.com/adfinis/Outdated/blob/4dc640e535ef9b001407cd8c1e03363e22dfbcd2/ember/app/components/maintainer-table/template.hbs#L12-L24